raster/r.colors.out/local_proto.h

```diff
@@ -0,0 +1,12 @@
+#include <grass/gis.h>
+#include <grass/parson.h>
+
+enum ColorFormat { RGB, HEX, HSV, XTERM };
+
+void write_json_rule(DCELL *val, DCELL *min, DCELL *max, int r, int g, int b,
+                     JSON_Array *root_array, int perc,
+                     enum ColorFormat clr_frmt, FILE *fp,
+                     JSON_Value *root_value);
+
```

Comment:
```suggestion
```

Only `print_json_colors()` needs prototype here, that is the only function accessed in the main files.

The other functions in `prt_json.c` can be declared `static`.